seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/utils/ExpressionUtils.java

```diff
@@ -199,6 +234,17 @@ public static Expression convert(
             }
             return Expressions.equal(column.getColumnName(), booleanExpression.isTrue());
         }
+        if (condition instanceof LikeExpression) {
+            LikeExpression expr = (LikeExpression) condition;
+            String columnName = ((Column) expr.getLeftExpression()).getColumnName();
+            String value = ((StringValue) expr.getRightExpression()).getValue();
+            LikeExpression.KeyWord keyWord = expr.getLikeKeyWord();
+            if (keyWord == LikeExpression.KeyWord.LIKE) {
+                return Expressions.startsWith(columnName, value);
```

Comment:
Can we not support like start with `%` now?

Reply:
sure, i will remove the logic about `like`

Reply:
We can keep it, and update the doc to tell user we can not support `SELECT * FROM t WHERE name LIKE '%John'` which like start with `%`.

Reply:
@litiliu

Reply:
added doc about the limitation of the supporting for like clause, @Hisoka-X  , PTAL again.